homeassistant/components/withings/__init__.py

```diff
@@ -44,12 +44,6 @@
                 vol.Required(CONF_CLIENT_ID): vol.All(cv.string, vol.Length(min=1)),
                 vol.Required(CONF_CLIENT_SECRET): vol.All(cv.string, vol.Length(min=1)),
                 vol.Optional(const.CONF_USE_WEBHOOK, default=False): cv.boolean,
-                vol.Required(const.CONF_PROFILES): vol.All(
```

Comment:
Removing this, causes a hard breaking change: The configuration becomes invalid if one has this in its configuration.

Another alternative approach: Mark `CONF_PROFILES` as deprecated. It would still be allowed in the config, would throw a warning in the logs and for the code, you can just ignore it.

Remove the deprecation notice after a couple of releases.